### ISSUE TITLE
Modified Localization temporarily gets sent to "undefined" group ... edit: when grouping attribute is set to visible=False

### ISSUE THREAD

**Issue by sbatchelder** [cvisionai/tator#894]
While reviewing a video with annotations, when modifying a localization (eg toggling an unrelated BOOL attribute) with class label grouping On, the localization gets moved to a "undefined" group. Furthermore, with Frame Seek Auto on, the frame doesn't change (the localization is still selected) but its entry on the right panel is suddenly/secretly moved to the new "undefined" group... creating the illusion that it wasn't updated (it's actually just the next localization which breaks the expected behavior of Frame Seek Auto). 

On page reload, the group class is not modified and the attribute is correctly updated, but there's a problem with interface and it's causing a workflow issue. 

Before "Verified" Toggled
![image](https://user-images.githubusercontent.com/44208509/195640560-a11cad43-43d0-40df-82eb-7418204e3170.png)

After "Verified" Toggled - note how the class count went down on center_spot_line and how undefined was created.
![image](https://user-images.githubusercontent.com/44208509/195640871-7e17ce6c-3609-43dc-bee6-09ad32a16d88.png)
![image](https://user-images.githubusercontent.com/44208509/195641067-80e89343-22e6-48fe-9b5c-b968e2649d95.png)

After page reload, interface displays correct group and updated value
![image](https://user-images.githubusercontent.com/44208509/195641369-db53459b-0b4a-4e2f-8f6e-02ae035d502f.png)

Possibly related to Issue #873 ?

Tator version 0.2.20

**Comment by sbatchelder** [comment]
This behavior was resolved. The grouping attribute "Class" was set to "visible=False". Jon helped with with an issue whereby I wasn't able to make changes to these attributes (we ended up using the django admin panel). When the grouping attribute was set back to visible=True, the issue of localizations being shunted to this "undefined" group no longer happened.

This may still count as a bug, but for the time being the immediate issue for me has been resolved.
